http-api/src/lib.rs

```diff
@@ -427,6 +443,51 @@ async fn blob_handler(
     Ok(warp::reply::json(&blob))
 }
 
+async fn patches_handler(ctx: Context, urn: Urn) -> Result<impl Reply, Rejection> {
+    let mut patches: Vec<Metadata> = Vec::new();
+    let repo = git2::Repository::open_bare(ctx.paths.git_dir()).map_err(Error::from)?;
+    let storage = ReadOnly::open(&ctx.paths).map_err(Error::from)?;
+    let info = project_info(urn.clone(), ctx.paths)?;
+    let peers = project::tracked(&info.meta, &storage)?;
+
+    // Iterating over all tracked peers, to get all possible patches
+    for peer in peers {
+        if let Ok(refs) = Refs::load(&storage, &urn, peer.id) {
+            let blobs = match refs {
+                Some(refs) => refs.tags().collect(),
+                None => vec![],
+            };
+            // Iterate over all refs, and return eventual tags aka patches
+            for blob in blobs {
+                if let Some(object) = storage.find_object(blob.1).map_err(Error::from)? {
+                    let tag = object.peel_to_tag().map_err(Error::from)?;
+                    // Only match tags that have a valid ut8 name that starts with patches/
+                    if !tag
+                        .name()
+                        .ok_or(Error::Utf8Error)?
+                        .starts_with("radicle-patch/")
```

Comment:
This doesn't match what the comment says. 🤔

Reply:
Thanks, slipped through the cracks!

Reply:
Thanks, slipped through the cracks!